posts/2020-11-20-JNDI-gRPC-200012.adoc

```diff
@@ -0,0 +1,389 @@
+---
+layout: post
+title: Support for gRPC, Java SE 15, and Custom JNDI for Enterpise Beans (EJBs) in Open Liberty 20.0.0.12
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/jakub-pomykala
+author_github: https://github.com/jakub-pomykala
+seo-title: Support for gRPC, Java SE 15, and Custom JNDI for Enterpise Beans (EJBs) in Open Liberty 20.0.0.12 - OpenLiberty.io
+seo-description: In Open Liberty 20.0.0.12 you can provide and consume gRPC services from your web applications. Java SE 15 is now in GA and we now offer support for Custom JDNI Names for Enterprise Beans(EJBs).
+blog_description: In Open Liberty 20.0.0.12 you can provide and consume gRPC services from your web applications. Java SE 15 is now in GA and we now offer support for Custom JDNI Names for Enterprise Beans(EJBs).
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+---
+= Support for gRPC, Java SE 15, and Custom JNDI for Enterpise Beans (EJBs) in Open Liberty 20.0.0.12
+Jakub Pomykala <https://github.com/jakub-pomykala>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+
+// tag::intro[]
+
+In Open Liberty 20.0.0.12 we now support gRPC 1.0 and gRPC Client 1.0 features. gRPC is a "high-performance, open source universal RPC framework." Another new supported feature is custom JNDI (Java Naming and Directory Interface) for Enterprise Beans (EJBs). This allows you to use custom JDNI names for looking up and/or injecting enterprise beans. Open Liberty 20.0.0.12 now also supports Java 15.
+
+In link:{url-about}[Open Liberty] 20.0.0.12:
+
+* <<grpc, Provide and consume gRPC services from your web applications (gRPC Service 1.0 and gRPC Client 1.0)>>
+* <<jndi, Support for Custom JNDI Names for Enterprise Beans (Jakarta Enterprise Beans 1.x, 2.x, 3.x)>>
+* <<java15, Support for Java SE 15>>
+* <<bugs, Significant bugs fixed in this release>>
```

Comment:
Did we purposefully remove the links to the sub-headings/sections for JNDI items (and gRPC)?  I'm not sure I have a preference with or without them, just trying to make sure it was a conscious decision.

Reply:
Yes, I spoke to @lauracowen and she recommended that we remove the since that list is only supposed to show the reader the main bits in the blog. With the subheadings it does a bit more than that and also crowds the section.